src/extension/loadLogs.ts

```diff
@@ -21,7 +21,7 @@ export async function loadLogs(uris: Uri[], token?: { isCancellationRequested: b
                 const file = fs.readFileSync(uri.fsPath, 'utf8')  // Assume scheme file.
                     .replace(/^\uFEFF/, ''); // Trim BOM.
                 const {data: log, pointers} = jsonMap.parse(file) as { data: Log, pointers: JsonMap};
-                log._uri = uri.toString();
+                log._uri = uri.scheme === 'file' ? uri.fsPath : uri.toString();
```

Comment:
There is a less invasive way to fix this issue than to change the underlying schema.

Reply:
Pushed a fix. Please let me know if it works